modules/db/src/main/scala/gem/dao/package.scala

```diff
@@ -75,19 +75,23 @@ package object dao extends MoreTupleOps with ToUserProgramRoleOps {
       EitherConnectionIO.right[A, T](c)
   }
 
-  // Angle mapping to signed arcseconds via NUMERIC. NOT implicit.
+  // Angle mapping to signed arcseconds via NUMERIC. NOT implicit. We're mapping a type that
+  // is six orders of magnitude more precise than the database column, so we will shift
+  // the decimal point back and forth.
```

Comment:
Yeah this isn't implicit so it's not the canonical mapping for angles. It's used explicitly by offsets so a human looking at the database will see a signed arcsecond value like `-1.25`. Same with the JSON mapping. I think as I build out the target model we'll get a better handle on how to map angles in a way that's precise but also human-readable, so I may revisit this.